reference-architecture/osp-cli/all-hosts.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+#
+# Execute these preparation steps on all hosts
+# Run on the bastion host
+#
+OCP_VERSION=${OCP_VERSION:-3.2}
```

Comment:
Currently defaults to 3.2 since that was the latest tested on OSP8 and reviewed in the refarch.  Will update with the next refarch release.